src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -563,311 +574,270 @@ const HomePage: React.FC = () => {
     permissionObj['resource']['resourceIdentifier'] = folderId
   }
 
-  const links: { url: string; label: string }[] = [
-    {
-      url: routes.toCustomDashboardHome({ accountId }),
-      label: 'Home'
-    },
-    {
-      url:
-        folderId === 'shared' ? routes.toCustomDashboardHome({ accountId }) : routes.toCustomFolderHome({ accountId }),
-      label: folderId === 'shared' ? getString('common.dashboards') : getString('dashboards.homePage.folders')
-    }
-  ]
-
-  if (folderId) {
-    links.push({
-      url: routes.toCustomDashboardHome({ accountId }),
-      label: title
-    })
-  }
+  React.useEffect(() => {
+    const links = getBreadcrumbLinks(folderDetail, accountId, getString('dashboards.homePage.folders'))
+    includeBreadcrumbs(links)
+  }, [folderDetail])
 
   return (
-    <Container className={css.customDashboards}>
-      <Page.Header
-        title={folderId !== 'shared' ? title : getString('common.dashboards')}
-        breadcrumbs={<Breadcrumbs links={links} />}
-        content={
-          <Layout.Horizontal spacing="medium">
-            <NavLink
-              className={css.tags}
-              activeClassName={css.activeTag}
-              to={routes.toCustomDashboardHome({ accountId, folderId })}
-            >
-              {getString('common.dashboards')}
-            </NavLink>
-            <NavLink className={css.tags} activeClassName={css.activeTag} to={routes.toCustomFolderHome({ accountId })}>
-              {getString('dashboards.homePage.folders')}
-            </NavLink>
-          </Layout.Horizontal>
-        }
-        toolbar={
-          <Text color={Color.PRIMARY_6} style={{ cursor: 'pointer' }} onClick={() => setDrawerOpen(true)}>
-            {' '}
-            <Icon name="question" /> {getString('getStarted')}
-          </Text>
-        }
-      />
-      <Page.Body
-        loading={loading || cloning}
-        retryOnError={() => {
-          return
-        }}
-        error={(error?.data as Error)?.message}
-      >
-        <GetStarted isOpen={isOpen} setDrawerOpen={val => setDrawerOpen(val)} />
-        <Layout.Vertical padding="large" background={Color.GREY_0}>
+    <Page.Body
+      loading={loading || cloning}
+      retryOnError={() => {
+        return
+      }}
+      error={(error?.data as Error)?.message}
+    >
+      <Layout.Vertical padding="large" background={Color.GREY_0}>
+        <Layout.Horizontal
+          style={{
+            borderBottom: '1px solid var(--grey-100)',
+            justifyContent: 'space-between',
+            paddingLeft: 0,
+            paddingRight: 0
+          }}
+          flex={true}
+        >
```

Comment:
Try to avoid inline styles; either use the props of the layout, or a class.
```suggestion
        <Layout.Horizontal border={{ bottom: true }} flex={{ justifyContent: 'space-between }}>
```